integ/scripts/bash/run-e2e-from-pipeline.sh

```
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -euo pipefail
+
+refreshcreds() {
+    echo "Refreshing credentials"
+    unset AWS_ACCESS_KEY_ID
+    unset AWS_SECRET_ACCESS_KEY
+    unset AWS_SESSION_TOKEN
+    creds=$(mktemp -d)/creds.json
+    AWS_STS_REGIONAL_ENDPOINTS=regional aws sts assume-role \
+        --role-arn "arn:aws:iam::050840627787:role/IntegrationTestRunner" \
+        --role-session-name "integration-tests" \
+        --external-id "7615a375-995f-40d1-978b-0dd479015922" > $creds
+    export AWS_ACCESS_KEY_ID="$(cat ${creds} | grep "AccessKeyId" | cut -d'"' -f 4)"
+    export AWS_SECRET_ACCESS_KEY="$(cat ${creds} | grep "SecretAccessKey" | cut -d'"' -f 4)"
+    export AWS_SESSION_TOKEN="$(cat ${creds} | grep "SessionToken" | cut -d'"' -f 4)"
+}
+
+scriptdir=$(cd $(dirname $0) && pwd)
+
+cd "$scriptdir"/../..
+
+yarn
+
+# Basic integ test configuration
+export SKIP_TEST_CONFIG=true
+export DEADLINE_VERSION="10.1.10.6"
+export USER_ACCEPTS_SSPL_FOR_RFDK_TESTS=true
```

We need to remove this since we don't want to accept Mongo SSPL license on anyone's behalf if they checkout this repository and try running this.

Instead, those building a pipeline that runs these tests should decide whether to accept the SSPL for themselves and configure this environment variable in their pipeline.

We could put a sanity check that happens here and fails fast if the environment variable is not set to accept the SSPL and display a user-friendly error message.

---

I've moved this SSPL acceptance out of here and injected it as an env var from the pipeline side.